lib/kwtool_web/controllers/auth_controller.ex

```diff
@@ -1,9 +1,25 @@
 defmodule KwtoolWeb.AuthController do
   use KwtoolWeb, :controller
 
+  alias Kwtool.Accounts
+  alias Kwtool.Accounts.Schemas.User
+
   plug :put_layout, "auth.html"
 
   def show(conn, _params) do
-    render(conn, "sign_up.html")
+    changeset = Accounts.change_user(%User{})
```

Comment:
Let's use registration changeset here
```suggestion
    changeset = User.registration_changeset()
```

Reply:
Updated